File: tests/data-clumps/test-cases/parameter-parameter/simple/positive/java/source/BillingProcessor.java
public class BillingProcessor {
    public void createInvoice(int patientId, int doctorId, boolean requiresFollowUp) {
        System.out.println("Creating invoice for patient " + patientId + " and doctor " + doctorId
                + " (requires follow-up: " + requiresFollowUp + ")");
    }
}

File: tests/data-clumps/test-cases/refactoring/parameter-parameter/simple/typescript/source-expected/BillingProcessor.ts
import { PatientIdDoctorIdRequiresFollowUpParams } from "./PatientIdDoctorIdRequiresFollowUpParams";

export class BillingProcessor {
  createInvoice(params: PatientIdDoctorIdRequiresFollowUpParams): void {
      const { patientId, doctorId, requiresFollowUp } = params;
    console.log(`Creating invoice for patient ${patientId} and doctor ${doctorId} (requires follow-up: ${requiresFollowUp})`);
  }
}
